ivy/functional/frontends/torch/comparison_ops.py

```diff
@@ -290,7 +290,7 @@ def topk(input, k, dim=None, largest=True, sorted=True, *, out=None):
 
 
 gt = greater
+ne = not_equal
 ge = greater_equal
 le = less_equal
 lt = less
-ne = not_equal
```

Comment:
I'm not sure why you made this change but it's causing lint errors, please revert it.